js/search-algorithm.js

```diff
@@ -136,7 +136,7 @@ define([
 
 		onDataReady: function() {
 			var config = Adapt.course.get("_search");
-			if (!config || !config._isEnabled ) return;
+			if (!config || _.has(config, '_isEnabled') && config._isEnabled === false) return;
```

Comment:
``config && _.has(config, '_isEnabled') && config._isEnabled === false`` has identical behaviour to ``config && config._isEnabled === false``